crates/update/spacetime-install.ps1

```diff
@@ -7,6 +7,30 @@ function Install {
     $ErrorActionPreference = 'Stop'
     [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
 
+    function CheckVCRuntime {
+        $paths = @(
+            "$env:SystemRoot\System32\vcruntime140.dll",
+            "$env:SystemRoot\SysWOW64\vcruntime140.dll"
+        )
+        foreach ($path in $paths) {
+            if (Test-Path $path) {
+                Write-Host "vcruntime140.dll is already installed at $path"
+                return $true
+            }
+        }
+        Write-Host "vcruntime140.dll is not installed"
+        return $false
+    }
+
+    if (-not (CheckVCRuntime)) {
+        $DownloadUrl = "https://aka.ms/vs/17/release/vc_redist.x64.exe"
+        Write-Output "Downloading vcruntime140.dll..."
+        $Installer = Join-Path ([System.IO.Path]::GetTempPath()) "vc_redist.x64.exe"
+        Invoke-WebRequest $DownloadUrl -OutFile $Installer -UseBasicParsing
+        Start-Process -Wait -FilePath $Installer -ArgumentList "/quiet", "/install"
+        return
```

Comment:
Not sure if this is intended or not but after the `vcruntime` install finishes this causes the script execution to end so then you have to run it again in order to install spacetimedb. I'll fix in a follow-up PR.